config/crd/bases/infrastructure.cluster.x-k8s.io_awsclusters.yaml

```diff
@@ -157,6 +157,11 @@ spec:
                 instanceState:
                   description: The current state of the instance.
                   type: string
+                networkInterfaces:
+                  description: Specifies ENIs to attach to instance
+                  items:
+                    type: string
+                  type: array
```

Comment:
`make generate` seems to not be putting the validation in for this part...

Reply:
The annotations are only defined on the types used by the AWSMachine, not the ones used by the AWSCluster

Reply:
This is because the AWSCluster object references the Instance struct to track the bastion host, I think it's fine to leave it this way, given that it's in status